lsst-epo/rubin-obs-client
# feat: support NOIRLab announcements

Resolves #598 

Add support for NOIRLab announcements to news pages and to the SWR data mixer. 

To test:
- (optional) pull corresponding API branch to get updated text in Craft
- Add a news post in Craft with the "Announcement" tag and an announcement ID
- See that the article appears on the client
- See that the article is searchable with title/image/date
- See that the article appears on homepage news grid
- Remove either the "Announcement" tag or the ID in Craft
- See that the post reverts to a Craft entry
